services/api/service.go

```diff
@@ -1563,6 +1563,38 @@ func (api *RelayAPI) checkSubmissionSlotDetails(w http.ResponseWriter, log *logr
 	return true
 }
 
+func (api *RelayAPI) checkBuilderEntry(w http.ResponseWriter, log *logrus.Entry, builderPubkey phase0.BLSPubKey) (*blockBuilderCacheEntry, bool) {
```

Comment:
quick function comment would be nice, in particular what's the bool return value